.eslintrc.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
                     ['~utils', './src/utils'],
                     ['~tests', './tests'],
                 ],
-                extensions: ['.js', '.ts'],
+                extensions: ['.js', '.ts', '.tsx'],
```

Comment:
This was needed to allow for our linting checks to run on a commit if a `.tsx` typescript-react file changed.